docs/development/pigs/pig-024.rst

```diff
@@ -0,0 +1,283 @@
+    one can precise:
+
+    "The author list has three parts: the authors that made significant contributions to the writing of the paper,
+    the members of the Gammapy Project Coordination Committee, and contributors to the Gammapy Project in alphabetical
+    order. The position in the author list does not correspond to contributions to the Gammapy Project as a whole. A more
+    complete list of contributors to the core package can be found in the package repository and at the Gammapy team
+    webpage."
+
+2. Corresponding author
+    It is the 'Gammapy Coordination Committee', associated with its usual mailing list (GAMMAPY-COORDINATION-L@IN2P3.FR).
+
+3. Acknowledgement
+    One has the freedom to precisely mention acknowledgements associated with the publication. In practice, it is
+    recommended to precise the grants or fellowships given to some authors. One should in any case always acknowledge
+    the Astropy project, to which we are affiliated, and our mandatory external libraries (e.g. numpy, scipy, matplotlib,
+    iminuit).
+
+Contribution in conferences
+-----------------------------
+
+The section is about any contribution in conferences (a talk, a poster and their associated proceedings) related to
+the Gammapy project itself. It does not concern a technical or scientific work that uses Gammapy as an open library.
+In this case, the citation scheme of Gammapy should be used by these authors.
+
+As the length of the author list is generally a constrain, the author list is reduced to the short list of contributors
+for the conference, followed by 'The Gammapy team' associated with a link to the Gammapy team webpage:
+
+ ::
+
+  oo, ff, tt, for `the Gammapy team <https://gammapy.org/team.html>`_
+
+If there is a corresponding author, the 'Gammapy Coordination Committee' associated with its usual mailing list, is
+used. Concerning the acknowledgement, the Astropy project should always be mentioned, and if possible our mandatory
+external libraries.
+
+
+4. Metadata files
+==================================
+
+Depending on the software repository, different metadata files are used in the eco-system of Open Source research
+software and are mandatory.
+
+The file ``codemeta.json`` is used by HAL and Software Heritage, and recommended by
+`ESCAPE <https://escape2020.pages.in2p3.fr/wp3/eossr/v0.3.2/metadata.html>`_ or
+`EOSC <https://www.eosc-pillar.eu/news/illustrating-benefits-eosc-research-communities-germany>`_.
+
+The file ``CITATION.cff`` is used by Zenodo and GitHub. Its format should follow the rules set up by the
+`Citation File Format project <https://pythonlang.dev/repo/citation-file-format-citation-file-format/>`_. A GitHub
+Action can be used to automatically check the compliance with the latest format. At the date of this PIG, there is no
+official scheme to handle current and past affiliations, that might be needed for LTS releases for example (see the
```

Comment:
I don't understand why an LTS release needs current or past affiliations. LTS releases are made from specific branches in the development scheme. Each of this branches needs to have the affiliation for that LTS release in the version of the `CITATION.cff` in that branch, then the tag (release) is made.


E.g. the workflow for 1.0 would be:

* Announce code-freeze
* Create the v1.0.x branch
* Ask people to update the CITATION.cff on the v1.0.x branch
* Make the v1.0.0 tag

Correct?

Reply:
I think the issue here is that there is no support for multiple affiliations in `CITATION.cff` (see https://github.com/citation-file-format/citation-file-format/issues/268), however for some cases it might be required, because contributors can be affiliated to multiple institutions at the same time. Or their Gammapy contributions were made while being affiliated to multiple institutes (which might be very common for post-docs, changing affiliations after 2 years).

I think the only simple solution here is that week ask people to put their "preferred" affiliation and make a choice. So if a person contributed over the time span of 2 years, which ends with an LTS release and changed affiliation in between then they have to specify which one they prefer to be listed e.g. on Zenodo. For associated papers their might be the possibility to extract the information (change of affiliation) from the git history and list multiple if needed.

Reply:
Citations in general don't really have this feature (bibtex doesn't even have a field for authors affiliations). So the only solution to this would be to ask people to contribute this information to the actual paper author list in the corresponding repo or collect this information via some survey for any specific paper.

This is what astropy did: call for paper authors to provide their information via a google form